include/megaapi.h

```diff
@@ -4137,6 +4309,15 @@ class MegaApi
         char* getStringHash(const char* base64pwkey, const char* email);
 
         /**
+         * @brief Get internal timestamp used by the SDK
+         *
+         * This is a time used in certain internal operations.
+         *
+         * @return actual SDK time in deciseconds
+         */
+        int getSDKtime();
```

Comment:
Waiter::ds is dstime (uint32_t) but I have changed it anyway because all other functions managing timestamps in MegaApi use "long long".